core/lib/zksync_core/src/eth_watch/tests.rs

```diff
@@ -648,31 +758,35 @@ fn upgrade_into_governor_log(upgrade: ProtocolUpgrade, eth_block: u64) -> Log {
     }
 }
 
+fn tx_into_token(tx: ProtocolUpgradeTx) -> Token {
+    Token::Tuple(vec![
```

Comment:
if we have such encodings - let's also include the link (or the name) of the solidity object (so that reader can easily see/verify )